### PR TITLE
Support custom working directory for bumpversion action

### PR DESCRIPTION
**Pull request by disrupted**
tested in the playground

### CODE REVIEW

File: actions/bump-version/action.yaml [hunk 1]
```diff
@@ -5,6 +5,10 @@ inputs:
   release-type:
     description: "The type of the release (major, minor or patch)."
     required: true
+  working-directory:
+    description: "The root directory of the poetry project."
```
Comment (yannick-roeder) [inline actions/bump-version/action.yaml:9]: I don't think this necessarily refers to a root of a poetry project. Just use a similar description like you did in the Readme

File: actions/bump-version/README.md [hunk 2]
```diff
@@ -1,56 +1,51 @@
...
+
+You need a `.bumpversion.cfg` file in `working-directory` (repository root by default). A minimal configuration could look like this:
+
 ```cfg
 [bumpversion]
 current_version = 0.0.1
-commit = True
-tag = True
+commit = False
+tag = False
```
Comment (yannick-roeder) [inline actions/bump-version/README.md:13]: Why did you change this?

Comment (disrupted) [reply actions/bump-version/README.md:13]: I thought it would be less confusing for the example in the readme, since this action doesn't do any of these things. I also added a disclaimer underneath to link to the `commit-and-push` action. Of course it can still be set to true. Are these two settings actually required? Otherwise I would just remove them altogether.

Comment (yannick-roeder) [reply actions/bump-version/README.md:13]: I'm not sure if they are required, but I think removing them is a good idea if you test that it still works then

Comment (disrupted) [reply actions/bump-version/README.md]: Can confirm, it works without those options set.

**Review by yannick-roeder — APPROVED** [review]
